site/docs/vendors.md

```diff
@@ -21,6 +21,12 @@ title: "Vendors"
 ## Vendors Supporting Iceberg Tables
 
 This page contains some of the vendors who are shipping and supporting Apache Iceberg in their products
+### [Bodo](https://bodo.ai)
```

Comment:
Please add a new line before this vendor mention.

Reply:
Added, thank you!